### DISCUSSION TITLE
[Feature Request] Multiple (chooseable) Tags on Home

### DISCUSSION THREAD

**Discussion by rauschgiftengel** [daledavies/jump#77] [Feature Requests]
First of all: Great tool. Switched from Dashy to Jump, as it is beatiful, and just enough to do the job. 

Is is possible to add a feature to have more than one tag showed on the front page ? Maybe let us choose wich ones are the additional tags ? 

Like the picture below (i´m a verry talented paint-artist ;)

![x](https://user-images.githubusercontent.com/59368208/213708622-b52171cc-daf3-422f-a943-7766745c1bc1.jpg)

**Comment by daledavies** [comment]
Hi thanks for reaching out, glad you like using Jump! :)

I'll consider this for a future enhancement but I do kind of like the simplicity of having just one tag per page, also this would require me to rethink the tags navigation.

Transferring this issue to a discussion.
